src/components/dynamic/user/Team.jsx

```diff
@@ -37,6 +37,10 @@ const Team = ({ user, setUser }) => {
   };
 
   const handleJoin = () => {
+    if (id.team.length === 0) {
```

Comment:
fixed

Reply:
u can just check for an empty string instead of character comparison